tests/installation/resolve_dependency_issues.pm

```diff
@@ -33,8 +33,12 @@ sub run {
 
 sub post_fail_hook {
     my $self = shift;
-    select_console 'root-console';
-    $self->upload_solvertestcase_logs();
```

Comment:
You shouldn't remove the cmd to upload solvertestcase, this will cause can't get the solvertestcase in the future in resolve_dependency_issues.

Reply:
In this case, when exiting from iso install/update, zypper is not available in single-user mode.

Reply:
I know this, so just add condition to avoid use zypper during installation while not simply remove it.

Reply:
done. thanks